### PR TITLE
Added titles to manager side pages and fixed grid for item details page

### PR DESCRIPTION
**Pull request by LawreenL**
**Finishing up for the Demo:**
- Added the titles "Reservations" and "Loans" to manager side pages. 
- Modified for loops to if statements so the "Reservations" title changes to "No Reservations" when empty (same with Loans). 
- Modified the grid layout for item details page to add padding on top and making it more responsive

*Loans page with active Loans*
screen shot 2018-10-28 at 5 54 17 pm
*Loans page with no active Loans*
screen shot 2018-10-28 at 5 54 31 pm
##
*Item Details page on desktop*
screen shot 2018-10-28 at 5 53 25 pm
*Item Details page on mobile*
screen shot 2018-10-28 at 5 53 50 pm

### CODE REVIEW

File: dam/loans/templates/loans/allReturns.html [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+                            <p>Reserved at: {{ loan.reserved_at }}</p>
+                            <p>Approved by: {{ loan.approved_by }}</p>
+                            <p>Approved at: {{ loan.approved_at }}</p>
+                        </div>
+                        <a class="link" href="{{ loan.pk }}"></a>
+                    </div>
+                </div>
+            {% endfor %}
+        {% else %}
             <h1 style="text-align: center;">No Loans</h1>
```
Comment (StephenOrJames) [inline dam/loans/templates/loans/allReturns.html]: Could you remove the alignment from this? All `h1` are aligned center from `style.css`.

File: dam/loans/templates/loans/allReservations.html [hunk 2]
```diff
@@ -2,22 +2,24 @@
...
+                            <p>Email: {{ reserve.client.email }}</p>
+                            <p>Name: {{ reserve.client.first_name|add:" "|add:reserve.client.last_name}}</p>
+                            <p>Reserved on: {{ reserve.reserved_at }}</p>
+                        </div>
+                        <a class="link" href="{{ reserve.pk }}"></a>
+                    </div>
+                </div>
+            {% endfor %}
+        {% else %}
             <h1 style="text-align: center;">No Reservations</h1>
```
Comment (StephenOrJames) [inline dam/loans/templates/loans/allReservations.html]: Could you remove the alignment from this? All `h1` are aligned center from `style.css`.

**Review by StephenOrJames — DISMISSED** [review]
A thought: would it be better to have the heading be the same, whether not there are items to be listed, and if there are no content then have an additional message underneath the title (rather than have the title change)? I'd personally prefer that (rather than just having a page with just a title), but up to you.

<hr>

Also, looks like the item detail page is slightly misaligned on mobile, and I assume that has to do with the rows and columns are nested (`row > row > col` vs `row > row > col > row` if I'm reading this correctly) since you added an "extra" row (there's already a row in `base.html`).

A solution might be to move the row in base out to each page individually (and maybe even the container), but that can be considered outside the scope of this fix unless you also want to handle that here as well.

image

**Comment by LawreenL** [comment]
Added Subtitles below bcs I can't decide but i think it looks good now.
screen shot 2018-10-28 at 7 03 16 pm

For the grid, ended up adding the top padding to base since it is used for every page and removed the top div from manager pages- leaving the grid alignment for @lawzeem

File: dam/loans/templates/loans/allReservations.html [hunk 3]
```diff
@@ -2,22 +2,23 @@
...
+                            <p>Name: {{ reserve.client.first_name|add:" "|add:reserve.client.last_name}}</p>
+                            <p>Reserved on: {{ reserve.reserved_at }}</p>
+                        </div>
+                        <a class="link" href="{{ reserve.pk }}"></a>
+                    </div>
+                </div>
+            {% endfor %}
+        {% else %}
+            <h1>Reservations</h1>
+            <h6 style="text-align: center;">No active reservations right now.</h6>
```
Comment (StephenOrJames) [inline dam/loans/templates/loans/allReservations.html:22]: Maybe `p` instead of `h6`?

File: dam/loans/templates/loans/allReturns.html [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
+                            <p>Approved by: {{ loan.approved_by }}</p>
+                            <p>Approved at: {{ loan.approved_at }}</p>
+                        </div>
+                        <a class="link" href="{{ loan.pk }}"></a>
+                    </div>
+                </div>
+            {% endfor %}
+        {% else %}
+            <h1>Loans</h1>
+            <h6 style="text-align: center;">No active loans right now.</h6>
```
Comment (StephenOrJames) [inline dam/loans/templates/loans/allReturns.html:23]: Maybe `p` instead of `h6`?

**Review by StephenOrJames — APPROVED** [review]
Excess whitespace at the top of pages that have a header is something to consider revisiting in the future (i.e. making it more consistent with those that don't have a header or something), but I'm okay with this.

**Pull request merged**